src/Tile_test.py

```diff
@@ -0,0 +1,66 @@
+from Tile import Tile
+from Terrain import Terrain
+from Building import Building
+from Faction import *
```

Comment:
This is a common thing to do in Java but is generally not preferred in Python because it makes dependencies less clear, has a performance overhead when this module is first loaded, and clutters the namespace (i.e. imports a bunch of extra variable names)

So even though it's quite verbose, I'd prefer listing these all out:
```suggestion
from Faction import (
    Dwarves,
    Engineers,
    Auren,
    Witches,
    Swarmlings,
    Mermaids,
    Alchemists,
    Darklings,
    Halflings,
    Cultists,
    Fakirs,
    Nomads,
    ChaosMagicians,
    Giants,
)
```